src/Blaster.WebApi/Features/Shared/_Layout.cshtml

```diff
@@ -43,22 +42,27 @@
                     <span aria-hidden="true"></span>
                     <span aria-hidden="true"></span>
                     <span aria-hidden="true"></span>
-                </a>        
+                </a>
             </div>
         </div>
-        
+
         <div id="menu" class="navbar-menu">
             <div class="navbar-start">
                 <a href="@Url.Action("Index", "Frontpage")" class="navbar-item">Home</a>
-                <a href="@Url.Action("Index", "Capability")" class="navbar-item">Capabilities</a>             
-                <a href="https://playbooks.dfds.cloud" class="navbar-item">Playbooks</a> 
+                <a href="@Url.Action("Index", "Capability")" class="navbar-item">Capabilities</a>
+                <a href="https://playbooks.dfds.cloud" class="navbar-item">Playbooks</a>
                 <a href="https://github.com/dfds" class="navbar-item">GitHub (Code examples)</a>
                 <a href="@Url.Action("SharedComponents", "Frontpage")" class="navbar-item">UI</a>
             </div>
 
             <div class="navbar-end">
-                <div class="navbar-item">
-                    @UserHelper.CurrentUser.Name
+                <div v-cloak class="navbar-item">
+                    <a v-if="isAuthenticated()" v-on:click="signOut()">
+                        {{getUserName() }}
+                    </a>
+                    <a v-if="!isAuthenticated()" v-on:click="signIn()">

```

Comment:
Login seems to be broken on any endpoint that isn't exactly '/', e.g. '/capabilities', '/capabilitydashboard?capabilityId=5'. Intended behaviour?

Reply:
Fixed problem. Apprently using postLogoutRedirectUri property of msal-vue is not a good idea. Using redirectUri instead fixed the problem :)